README.md

```diff
@@ -1 +1,87 @@
-# youtube-search-api
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```

Comment:
```
npm start
```
is more readable format for Readme
It also has the copy click button by default